codalab/apps/authenz/forms.py

```diff
@@ -7,7 +8,8 @@ class CodalabSignupForm(forms.Form):
     """
     team_name = forms.CharField(max_length=64, required=False)
     contact_email = forms.EmailField(required=False)
-    newsletter_opt_in = forms.BooleanField(initial=False, required=False)
+    if settings.MAILCHIMP_API_KEY:
+        newsletter_opt_in = forms.BooleanField(initial=False, required=False)
```

Comment:
Should probably not do it like this, instead make it default to False and not required, hide the field on the frontend?

Reply:
All fields are now hidden on the front end via context processor.